public/locale/en.json

```diff
@@ -1040,6 +1041,7 @@
   "not_specified": "Not Specified",
   "notes": "Notes",
   "notes_placeholder": "Type your Notes",
+  "nothing_to_copy": "Nothing To Copy here!",
```

Comment:
remove unused translation keys

Reply:
@rithviknishad made the required changes.